client.go

```diff
@@ -0,0 +1,133 @@
+package main
```

Comment:
¿no sería mejor crear su propio paquete? Para que no todo este cargado en main y solo se importe donde se ocupe

Reply:
Closing this as is a duplicated from previous conversation.